container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -561,13 +442,7 @@ func (manager *KubernetesManager) CreatePersistentVolumeClaim(
 		return nil, stacktrace.Propagate(err, "Failed to create volume claim '%s'", volumeClaimName)
 	}
 
-	// Wait for the PVC to become bound and return that object (which will have VolumeName filled out)
-	boundVolumeClaim, err := manager.waitForPersistentVolumeClaimBinding(ctx, namespace, volumeClaim.Name)
```

Comment:
why you are removing this? is it, not usable to determine that the volume is created an can be used?

Reply:
@leoporoli We are using dynamic provisioning

So I create PVC
I start pod with the PVC mount;

This is when the PV will automatically be created by k8s; so the claim will never be bound; as this validation would happen before the service is created

Reply:
The service will fail to start if the PV doesn't get created